app/crud.py

```diff
@@ -330,6 +330,13 @@ def delete_price(db: Session, db_price: PriceFull) -> bool:
     db.commit()
     return True
 
+def update_price(db: Session, price: Price, new_values: dict):
+    for key, value in new_values.items():
```

Comment:
so here we can update every field from the `PriceCreateWithValidation` schema ?
- maybe add a typing on the `new_values` parameter ?
- what if we set a new proof_id, that does not belong to the user, would it work ? (it shouldn't)
- what if we sent a new product_code ? would the new price be linked to the new product ? would the corresponding product price_count be updated ?

Reply:
as discussed in the corresponding issue - #181 - i think it would be best for now to restrict the list of editable fields, fully test it, and then opening it up slowly to other more complex fields

Reply:
Okay ! Fornow I will create a basic_updateable_type and add it !

Reply:
great that you added the `PriceBasicUpdatableFields` :+1: 

one last thing : could you add a test trying to change a field not in PriceBasicUpdatableFields. what's the expected behavior ? silent update without change ? or a pydantic error ? for instance what I mentioned above : changing the proof_id

Reply:
yess wiil add !